src/components/ComboBox/sgds-combo-box.ts

```diff
@@ -66,11 +70,35 @@ export class SgdsComboBox extends SelectElement {
   @state() private optionList: SgdsComboBoxOptionData[] = [];
   @state() private emptyMenu = false;
 
+  // Used to show and hide the clear button
+  @state() isFocused = false;
```

Comment:
```suggestion
  @state() private isFocused = false;
```

so that it wont appear in API table in storybook